core/mutator/entry/entry.go

```diff
@@ -81,20 +76,20 @@ func (*Entry) Mutate(oldValue, mutation []byte) ([]byte, error) {
 		return nil, mutator.ErrMissingKey
 	}
 
-	if err := verifyKeys(oldValue, kv, update, entry); err != nil {
+	if err := VerifyKeys(oldValue, kv, update, entry); err != nil {
```

Comment:
I couldn't find a new external caller to `verifyKeys`.  Perhaps I'm missing something? 
Would love to keep it it a private API if it's not used externally.